lib/controllers/profiles.js

```diff
@@ -3,4 +3,11 @@ const ProfileService = require('../services/ProfileService');
 
 module.exports = Router().post('/', async (req, res, next) => {
   // TODO: Implement me!
+  try {
+    const profile = await ProfileService.create(req.body);
+    res.send(profile);
+  } catch (error) {
+    error.status = 404;
```

Comment:
We should only send 404 errors in specific cases, usually when something isn't found